pages/1.12/release-notes/1.12.1/index.md

```diff
@@ -0,0 +1,169 @@
+---
+layout: layout.pug
+navigationTitle: Release Notes for 1.12.1
+title: Release Notes for 1.12.1
+menuWeight: 5
+excerpt: Release notes for DC/OS 1.12.1
+---
+
+DC/OS Version 1.12.1 - Released 3 January 2019
```

Comment:
Use the `Date` format as listed in previous RNs. Example: https://docs.mesosphere.com/1.12/release-notes/1.12.0-beta1/

Reply:
Done